vocabularies/Common.xml

```diff
@@ -1063,6 +1063,9 @@ only `TargetProperties` and `TargetEntities` are relevant. They are addressed vi
         <Property Name="SourceEntities" Type="Collection(Edm.NavigationPropertyPath)" Nullable="false">
           <Annotation Term="Core.Description" String="Changes to one or more of these entities may affect the targets. An empty path means the annotation target." />
         </Property>
+        <Property Name="SourceAnnotations" Type="Collection(Edm.AnnotationPath)" Nullable="false">
+          <Annotation Term="Core.Description" String="Each `Edm.AnnotationPath` must be valid for [`UI.ReferenceFacet/Target`](UI.md#ReferenceFacet). Changes to the values of one or more of the referenced structural properties may affect the targets." />

```

Comment:
Can you please explain in the LongDescription of the complex type how these will be used? After all annotations don't change in $metadata.

Reply:
No quite sure whether my change addresses what you meant. Please check again.